src/llmcompressor/modifiers/utils/compression_wrapper.py

```diff
@@ -1,117 +0,0 @@
-from abc import ABC, abstractmethod
-from typing import Optional, Set
-
-import torch
-import torch.nn as nn
-from torch.nn import Module
-
-try:
-    import transformers
-except ImportError as err:
-    transformers = None
-    transformers_err = err
-
-torch.backends.cuda.matmul.allow_tf32 = False
-torch.backends.cudnn.allow_tf32 = False
```

Comment:
@rahul-tuli do you know why these were set to False?

Reply:
I believe this was related to supporting mixed precision forward passes, which we no longer concern ourselves with